CorsixTH/Lua/hospital.lua

```diff
@@ -2440,3 +2442,10 @@ end
 function Hospital:announceStaffLeave()
   -- Nothing to do, override in a derived class.
 end
+
+--! Reset the death counts (key losing conditions)
+function Hospital:resetDeathCount()
+  self.num_deaths = 0
+  self.num_deaths_this_year = 0
+  self.statistics.deaths = 0
+end
```

Comment:
Based on previous cheat code -- we usually only elect to use a function from another class if it is used routinely.
In this case, it's a method specific to the cheat itself.